colour/models/din99.py

```diff
@@ -85,37 +100,41 @@ def Lab_to_DIN99(Lab, k_E=1, k_CH=1):
     >>> Lab_to_DIN99(Lab)  # doctest: +ELLIPSIS
     array([ 53.2282198...,  28.4163465...,   3.8983955...])
     """
+    p = DIN99_METHODS[str(method)]
```

Comment:
We should validate the method: `p = DIN99_METHODS[validate_method(method, DIN99_METHODS)]`

Maybe `p` should actually be `c`, standing for coefficient and maybe we expand them too, instead of accessing them:

```python
c_1, c_2, c_3, c_4, c_5, c_6, c_7, c_8 = DIN99_METHODS[validate_method(method, DIN99_METHODS)]
```